### PR TITLE
add delete round

### PR DESCRIPTION
**Pull request by SvanaBirgis**
Issue #6

### CODE REVIEW

File: app/src/main/java/hugbo/golfskor/network/NetworkService.kt [hunk 1]
```diff
@@ -68,6 +75,19 @@ interface NetworkService {
...
+        @Header("Authorization") authToken: String,
+        @Query("roundId") roundId: Int,
+        @Query("userId") userId: Int
+    ): Response<Void>
+
+    //@HTTP(method = "DELETE", path = "/api/round", hasBody = true)
+    //suspend fun deleteRound(
+    //    @Header("Authorization") authToken: String,
+    //    @Body roundId: Int
+    //): Response<Void>
```
Comment (ofurtumi) [inline app/src/main/java/hugbo/golfskor/network/NetworkService.kt]: Taka út

File: app/src/main/java/hugbo/golfskor/ui/viewModels/ProfileViewModel.kt [hunk 2]
```diff
@@ -75,4 +77,27 @@ class ProfileViewModel(savedStateHandle: SavedStateHandle) : ViewModel() {
...
+
+            profileUiState = try {
+                Log.d("Deleting by roundId",  "$roundId")
+                println("asdf")
+                 val result = GolfSkorApi.retrofitService.deleteRound(
+                     "Bearer $authToken",
+                     roundId,
+                     userId,
+                 )
+                // TODO eyda round ur oldstate og skila
```
Comment (ofurtumi) [inline app/src/main/java/hugbo/golfskor/ui/viewModels/ProfileViewModel.kt]: Taka út

**Review by ofurtumi — APPROVED** [review]
Smá comment en annar guddy

**Review by Gunnarbjo — APPROVED** [review]
Looks good

**Pull request merged**